taxi/tests/test_views.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Car, Manufacturer
+
+URL = reverse("taxi:car-list")
+
+
+class PublicLoginRequireTest(TestCase):
+    def test_login_required(self):
+        response = self.client.get(URL)
+        self.assertNotEquals(response.status_code, 200)
```

Comment:
Consider using `self.assertNotEqual` instead of `self.assertNotEquals` for consistency with Django's testing framework. `assertNotEqual` is the preferred method name.